core/input/input.h

```diff
@@ -79,6 +79,15 @@ class Input : public Object {
 		CURSOR_MAX
 	};
 
+	class JoypadFeatures {
+	public:
+		virtual ~JoypadFeatures() {}
+
+		// None at the moment, but later we can add new features like:
+		// virtual bool has_joy_accelerometer() const { return false; }
+		// virtual bool set_joy_accelerometer_enabled(bool p_enable) { return false; }
```

Comment:
```suggestion
		//virtual bool has_joy_accelerometer() const { return false; }
		//virtual bool set_joy_accelerometer_enabled(bool p_enable) { return false; }
```
Commented out code has no leading space for readability